js/menumobile.js

```diff
@@ -0,0 +1,13 @@
+const menuBur = document.getElementById('burguer-menu');
+const slideMenu = document.getElementById('slidemenu');
+const elemMenu = document.querySelectorAll('#slidemenu a');
+
+function toggleMenu() {
+  slideMenu.classList.toggle('active');
+}
```

Comment:
[OPTIONAL ] Please use the `ES6 arrow function` because `arrow functions` are more like `function` statements, except that they bind this to the `parent scope`. If the arrow function is in the top scope, this argument will refer to the `window/global` scope, while an arrow function inside a `regular` function will have this argument the same as its `outer` function.

```suggestion
const toggleMenu = () => slideMenu.classList.toggle('active');
```

Reply:
Thankyou. I plan to use this function in next feautures, for that reason I wolud leave it is this way.